### ISSUE TITLE
When can we get  Integration with Supabase.Realtime Realtime listeners for database changes feature?

### ISSUE THREAD

**Issue by BenCherif** [supabase-community/supabase-go#20]
# Feature request

## Integration with [Supabase.Realtime](https://github.com/supabase-community/realtime-go)
Realtime listeners for database changes

**Comment by ZanzyTHEbar** [comment]
Indeed, this feature is long overdue and a version by the community has been available (though needs to be updated) for awhile now. <https://github.com/overseedio/realtime-go>
